Automattic/vip-go-mu-plugins
# VaultPress 2.1.3

Details here: https://github.com/Automattic/vaultpress/blob/e12f3b499791cb8bee8a27044371e1dcaf57dde7/readme.txt#L60-L68

```
** Compatibility **

* General: Handle new user meta actions.
* General: Implement the spread operator for our IXR class methods to match WordPress Core behavior in 5.5.
* General: Update some miscellaneous dependencies.
```

Note: still needs testing.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Pull down PR
1. Poke around the site and make sure all base features continue to work.
